README.md

```diff
@@ -22,7 +22,7 @@ near login
 click the link and create your own contract ID
 
 Step 2:
-Modify src/config.js line that sets the contractName. Set it to id from step 1.
+Modify src/config.js line that sets the contractName. Set it to id from step 1. Note that if you are on testnet, please write "corgis" instead of "corgis.testnet".
```

Comment:
I test this but not working, when I use `sushi.testnet` logged in, I have to use `sushi.testnet` instead of `sushi`

Reply:
this is weird, for me, it only worked when I wrote anaiscorgis instead of anaiscorgis.testnet